src/core-sdk-tasks/core-sdk-tasks.csproj

```diff
@@ -15,7 +15,8 @@
     <PackageReference Include="NuGet.Versioning" Version="$(NuGetVersioningPackageVersion)" />
     <PackageReference Include="NuGet.Packaging" Version="$(NuGetVersioningPackageVersion)" />
     <PackageReference Include="System.Reflection.Metadata" Version="1.4.2" />
-    <PackageReference Include="WindowsAzure.Storage" Version="8.4.0" Condition="'$(DotNetBuildFromSource)' != 'true'" />
+    <PackageReference Include="WindowsAzure.Storage" Version="9.3.3" Condition="'$(DotNetBuildFromSource)' != 'true'" />
```

Comment:
Any reason why these versions are hard coded and not under eng\versions.props?

Reply:
The versions were previously hard coded, I'm not sure of the reasoning. Do you think they should not be?

Reply:
Not necessarily, but since we have code flow, if these values were to change in other repos that flow to the installer, we'd be getting the changes automatically.